src/main/webapp/dev/js/main/instructorFeedbackEdit.js

```diff
@@ -519,6 +531,8 @@ function enableNewQuestion() {
         /* eslint-enable camelcase */
     }
 
+    removeRequiredIfElementHidden(); // TODO: Remove this function call when #8688 is fixed.
```

Comment:
Note that we are adding a new click handler everytime the user clicks the add question button, not a major issue and we are going to get rid of the workaround later on anyway.